archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/util/InputTree.java

```diff
@@ -144,6 +144,9 @@ private void addNodes(Node node) {
         node.children().forEach(this::addNodes);
     }
 
+    /**
```

Comment:
Sorry you had to do all this JavaDoc; how did this get past checkstyle?

Reply:
I have no idea how that fell through the cracks, this is weird.